### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by wathigo**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,1011 @@
...
+.article .author p a:hover{
+  color: #fff;
+}
+.article .author p i{
+  padding: 0 3px 0 6px;
+}
+.latest-news-wrapper, .latest-deals-wrapper{
+  width: 90%;
+  margin: 0 auto 30px auto;
+  /* display: flex;
```
Comment (JamesOkunlade) [inline css/styles.css]: Remove these comments. Don't leave unused styles in your file as comments.

**Review by JamesOkunlade — APPROVED** [review]
### Hello guys!! 

You did a very good job with the TNW clone. Your HTML is well indented and contains no error and it renders very well across multiple browsers.  

However, I just noticed the commits are not evenly distributed between you two. You should ensure on this when working on your next collaborative project. Also, I left a comment on a line in your CSS, refactor as I have instructed. 

 When you're done, merge to master, close pull request and fill the project completion form.

Good luck with your future projects :+1::+1:

**Comment by wathigo** [comment]
Thank you @JamesOkunlade

**Comment by sonegillis** [comment]
Thanks @JamesOkunlade

**Pull request merged**